wgpu-core/src/lib.rs

```diff
@@ -31,6 +31,7 @@ pub mod device;
 pub mod hub;
 pub mod id;
 pub mod instance;
+pub mod logging;
```

Comment:
None of the other modules we have ends with "ing". Could this one be just `log`?

Reply:
I named it such to avoid conflict with `log`.

Reply:
Yup!